keyboards/crkbd/keymaps/ninjonas/config.h

```diff
@@ -0,0 +1,48 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#define USE_SERIAL_PD2
+
+#define TAPPING_FORCE_HOLD
+
+#ifdef RGBLIGHT_ENABLE
```

Comment:
You may want to consider using the RGB Matrix feature over the RGB Light feature, since it works better, with how the keyboard is set up.

Reply:
Thank you for the recommendation. I tried the RGB Matrix before and it failed on me. Let me get back on this one as I'll need to read the docs and will create a new PR when I'm ready.